### DISCUSSION TITLE
Is it possible to target function expressions in `padding-line-between-statements`?

### DISCUSSION THREAD

**Discussion by slifty** [eslint/eslint#15402] [Q&A]
I have code that defines a series of function expressions in a row.

```
const x = 15

const foo = () => { ... }

const bar = () => { ... }

const baz = () => { ... }
```

I would like to require padding between ALL of those function expressions (e.g. both before and after each function expression), but I don't necessarily want padding around other `block-like` elements (e.g. if statements); I also don't necessarily want to force padding on `multiline-const` items in general.  It seems like this maybe isn't possible, but I figure this is the right place to ask!

(And if it isn't possible, might the project consider adding that type?)

For context, with functional react components I'm defining my handlers as function expressions rather than class elements.  This means I'm losing my linting protection around padding, and would *love* a way to keep it.

**Comment by flyingangel** [comment]
As for me, I need a seperation of normal `const` and the arrow function as well. Ex

    const { productDispatch, productState } = useContext(stateContext);
    const handleAddToCart = (product) => () => dispatch({
        payload: product,
        type: 'add'
    });

I need an automatic padding between these two
